dev-tools/maintenance/cloc-report.py

```
@@ -0,0 +1,744 @@
+    if not values:
+        return ["summary"]
+    if values[0].startswith("-") and values[0] not in {"-h", "--help"}:
+        return ["summary", *values]
+    return values
+
+
+def print_json(payload: object) -> None:
+    print(json.dumps(payload, indent=2))
+
+
+def main(argv: Sequence[str] = sys.argv[1:]) -> int:
```

Ruff (0.15.21)

[warning] Too many branches (18 > 12)

(PLR0912)